### PR TITLE
feat: add about page to the uhi-frontend

### PR DESCRIPTION
**Pull request by xentro-sam**
Complete about page with drop down menu and routing.

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                </div>
+
+                <ul className="hidden md:flex space-x-10 text-xl  font-myText">
+                    <li className='hover:font-extrabold hover:underline'>Home</li>
+                    <li className='hover:font-extrabold hover:underline'>Documentation</li>
+                    <li className='hover:font-extrabold hover:underline'>FAQ&apos;s</li>
+                    <li className='hover:font-extrabold hover:underline'>Contact Us</li>
+                    <li className='hover:font-extrabold hover:underline'>Login</li>
+                </ul>
+                {/* Hamburger */}
```
Comment (Saatwik-S) [inline src/components/Navbar.jsx:25]: remove the commented line

File: src/components/Dropdown.jsx [hunk 2]
```diff
@@ -27,41 +28,41 @@ export default function Dropdown() {
...
-                                </a>
+                                </Link>
                             )}
                         </Menu.Item>
                         <Menu.Item>
                             {({ active }) => (
-                                <a
-                                    href="#"
+                                <Link
+                                    href="/about/Citizens"
```
Comment (Saatwik-S) [inline src/components/Dropdown.jsx:58]: remove capital letters in routes ( change the file names Citizens.jsx -> citizens.jsx ) etc

File: src/components/Dropdown.jsx [hunk 3]
```diff
@@ -27,41 +28,41 @@ export default function Dropdown() {
...
-                                </a>
+                                </Link>
                             )}
                         </Menu.Item>
                         <Menu.Item>
                             {({ active }) => (
-                                <a
-                                    href="#"
+                                <Link
+                                    href="/about/Doctors"
```
Comment (Saatwik-S) [inline src/components/Dropdown.jsx:45]: remove caps in. routes

**Review by Saatwik-S — CHANGES_REQUESTED** [review]
Change the file naming structures.

File: src/components/Dropdown.jsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+function classNames(...classes) {
+    return classes.filter(Boolean).join(' ')
+}
+
+export default function Dropdown() {
+    return (
+
+        <Menu as="div" className="relative inline-block text-left">
+            <div>
+                <Menu.Button className="mx-auto hidden md:flex space-x-10 hover:font-extrabold hover:underline w-full justify-center rounded-md bg-uhigreen px-4 font-medium text-black hover:bg-uhigreen-secondary focus:outline-none focus:ring-2 focus:ring-uhigreen focus:ring-offset-2 focus:ring-offset-uhigreen">
```
Comment (rkdixit96) [inline src/components/Dropdown.jsx:14]: use classNames here
```
classNames(
    class1 class3,
    class 2
)
```

File: src/components/DoctorsHeader.jsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export default function DoctorsHeader() {
+    return (
+        <div className="w-full bg-uhigreen-secondary">
+            <div className='container mx-auto py-5 flex justify-between items-center'>
+                <h1>DOCTORS</h1>
+            </div>
+        </div>
+    )
+}
```
Comment (rkdixit96) [inline src/components/DoctorsHeader.jsx:9]: Combine these to a single component - Pass a prop

File: src/components/DoctorsInfo.jsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+        <div className='container mx-auto py-5 justify-between flex flex-col'>
+            <div className='flex flex-col items-center justify-center'>
+                <p className='text-xl'>The ABDM has built verified registries for healthcare facilities and professionals who are given unique and trustable identity, which can facilitate ease of discovery and better access to care. Health professionals will have access to patient's longitudinal health history with their consent and can ensure follow-ups and plan treatment using telehealth solutions under the Unified Health Interface (UHI), ensuring quality and affordable health services. The building blocks of ABHA and the key registries - Health Professionals Registry (HPR), Health Facility Registry (HFR), and UHI, are built on a robust layer of Digital Public Goods like Aadhaar and UPI which introduces reliability and transparency to the system.</p>
+            </div>
+            <div className='flex flex-col items-center justify-center'>
+                <Image src={infoDiagram} alt='' className='h-1/4 w-1/4'></Image>
+            </div>
+        </div>
+    )
+}
```
Comment (rkdixit96) [inline src/components/DoctorsInfo.jsx:15]: Combine into a component and pass a prop

File: src/components/DoctorsInfo.jsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import Image from 'next/image'
+import infoDiagram from '@/assets/citizens_doctors_info_diagram.png'
+
+export default function DoctorsInfo() {
+    return (
+        <div className='container mx-auto py-5 justify-between flex flex-col'>
+            <div className='flex flex-col items-center justify-center'>
+                <p className='text-xl'>The ABDM has built verified registries for healthcare facilities and professionals who are given unique and trustable identity, which can facilitate ease of discovery and better access to care. Health professionals will have access to patient's longitudinal health history with their consent and can ensure follow-ups and plan treatment using telehealth solutions under the Unified Health Interface (UHI), ensuring quality and affordable health services. The building blocks of ABHA and the key registries - Health Professionals Registry (HPR), Health Facility Registry (HFR), and UHI, are built on a robust layer of Digital Public Goods like Aadhaar and UPI which introduces reliability and transparency to the system.</p>
```
Comment (rkdixit96) [inline src/components/DoctorsInfo.jsx:8]: Create a file called screen text and import it as variable here

Comment (rkdixit96) [reply src/components/DoctorsInfo.jsx:8]:
```
{
    about:
        doctor:
            adsfasdfasdf
   login:
       asdfadsf
}
```

File: src/components/Footer.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+                <div>
+                    <h2 className="mb-6 text-xl font-sans">Patient Locker</h2>
+                    <ul className="textß-m hidden md:block">
+                        <li className="mb-4">
+                            <a className=" hover:underline">
+                                <Image src={qr} alt='' className='h-12 w-12'></Image>
+                            </a>
+                        </li>
+
```
Comment (rkdixit96) [inline src/components/Footer.jsx:51]: All text to screen text file

**Review by rkdixit96 — CHANGES_REQUESTED** [review]

**Pull request merged**